app/models/folder.rb

```diff
@@ -5,6 +5,9 @@
 # Folder model
 class Folder < Common::Base
   include ActiveModel::Validations
+  include RedisCaching
```

Comment:
This ends up being a nice light weight way to add caching without affecting all `Common::Base` children 👍